src/Core/Managed/Shared/Extensibility/Implementation/Metrics/MetricTerms.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Metrics
+{
+    using System;
+
+    internal static class MetricTerms
+    {
+        private const string MetricPropertiesNamePrefix = "Microsoft.ApplicationInsights.Metrics";
+
+        public static class Aggregation
+        {
+            public static class Interval
+            {
+                public static class Moniker
+                {
+                    public const string Key = MetricPropertiesNamePrefix + ".Aggregation.IntervalMs";
+                }
+            }
+        }
+
+        public static class Extraction
+        {
+            public static class ProcessedByExtractors
+            {
+                public static class Moniker
+                {
+                    public const string Key = MetricPropertiesNamePrefix + ".Extraction.ProcessedByExtractors";
+                    public const string ExtractorInfoTemplate = "(Name:{0}, Ver:{1})";      // $"(Name:{ExtractorName}, Ver:{ExtractorVersion})"
+                }
+            }
+        }
+
+        public static class Autocollection
+        {
+            public static class Moniker
+            {
+                public const string Key = MetricPropertiesNamePrefix + ".MetricIsAutocollected";
```

Comment:
In the situation with drops there is always the last drop. Currently the billing size of the metric is about 600 bytes. The string of 50 characters increases the metric size by 8.33%, rather a large drop.